plugins/+matlab/+ciplugins/+github/getDefaultPlugins.m

```diff
@@ -0,0 +1,13 @@
+function plugins = getDefaultPlugins(pluginProviderData)
```

Comment:
The `matlab` namespace is reserved for stuff that ships with MATLAB. Like in Jenkins, this namespace should be `ciplugins.github` rather than `matlab.ciplugins.github`.